extensions/blocks/podcast-player/style.scss

```diff
@@ -181,13 +187,17 @@ $block-border-color: $dark-gray-100;
 	&.has-secondary .jetpack-podcast-player__episode-link {
 		color: inherit;
 	}
+<<<<<<< HEAD
```

Comment:
Looks like something went wrong during the rebase.

Reply:
Rebased again and squashed rebase commits. It should be good to go once the tests pass.